### ISSUE TITLE
[A11y] Desktop/Mobile Search Results - Grid: Image Thumbnail Labels

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1816]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Search

### Admin user?

No

### Issue Description

Desktop/Mobile Search Results - Grid: https://staging.oregondigital.org/catalog?locale=en&q%5B%5D=test&search_field=all_fields&utf8=%E2%9C%93&view=gallery

The image thumbnails have the following failed instance:
label: Ensures every form element has a label

Image 1
Path: #batch_document_v979v313f
Snippet:
`<input type="checkbox" onfocus="setGalleryCheckbox(this)" onblur="setGalleryCheckbox(this)" onchange="setGalleryCheckbox(this)" name="batch_document_ids[]" id="batch_document_v979v313f" value="v979v313f" class="batch_document_selector">`

Image 2
Path: #batch_document_9z902z939
Snippet:
`<input type="checkbox" onfocus="setGalleryCheckbox(this)" onblur="setGalleryCheckbox(this)" onchange="setGalleryCheckbox(this)" name="batch_document_ids[]" id="batch_document_9z902z939" value="9z902z939" class="batch_document_selector">`

Image 4
Path: #batch_document_kw52j8067
Snippet:
`<input type="checkbox" onfocus="setGalleryCheckbox(this)" onblur="setGalleryCheckbox(this)" onchange="setGalleryCheckbox(this)" name="batch_document_ids[]" id="batch_document_kw52j8067" value="kw52j8067" class="batch_document_selector">`

Image 7
Path: #batch_document_1544bp11q
Snippet:
`<input type="checkbox" onfocus="setGalleryCheckbox(this)" onblur="setGalleryCheckbox(this)" onchange="setGalleryCheckbox(this)" name="batch_document_ids[]" id="batch_document_1544bp11q" value="1544bp11q" class="batch_document_selector">`

### Suggested Fix

Fix ONE of the following:

- Form element does not have an implicit (wrapped) <label>
- Form element does not have an explicit <label>
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute
- Element has no placeholder attribute
- Element's default semantics were not overridden with role="none" or role="presentation"

**Comment by azle** [comment]
Above issue is fixed. But a new issue popped up when validating page see below:

<img width="1614" alt="Screen Shot 2021-12-07 at 9 20 06 AM" src="https://user-images.githubusercontent.com/8353860/145076075-c983bb21-4575-46a0-9287-0a5d28ef653e.png">
